src/lib/components/button.svelte

```diff
@@ -12,15 +12,16 @@
     on:click
     class:selected={active}
     {disabled}
-    class={`${size} ${variant}`}><slot /></button
+    class={`flex items-center justify-center ${size} ${variant}`}
+    ><slot /></button
   >
 {:else}
   <a
     {href}
     on:click
     class:selected={active}
     {disabled}
-    class={`${size} ${variant}`}><slot /></a
+    class={`flex items-center justify-center ${size} ${variant}`}><slot /></a
```

Comment:
This is duplicated code, you might want to just store all of these attributes in an object and [then spread them across](https://svelte.dev/tutorial/spread-props) both the `a` and the `button`, that way we're not implementing everything twice.